filecoin-proofs/src/api/update.rs

```diff
@@ -272,6 +277,9 @@ pub fn remove_encoded_data<Tree: 'static + MerkleTreeTrait<Hasher = TreeRHasher>
     info!("remove_data:start");
 
     let p_aux = util::get_p_aux::<Tree>(replica_cache_path)?;
+    #[cfg(feature = "fixed-rows-to-discard")]
```

Comment:
In order to clean-up all of the switching around this, why not just update the `get_t_aux` method to take the sector_bytes and ignore it inside if not needed (using the switching)?

Reply:
Good idea!